### DISCUSSION TITLE
How to configure a user-defined tree directory structure

### DISCUSSION THREAD

**Discussion by know-that** [opcodesio/log-viewer#129] [Q&A] [answered]
like this

![image](https://user-images.githubusercontent.com/31246220/195259476-ba1b1c07-ffa8-4e69-90cc-469e86eb526e.png)

No tree like directory structure is displayed on the page
![image](https://user-images.githubusercontent.com/31246220/195259501-5f65d8b2-0bf9-4714-abf7-7f0205be4053.png)

**Comment by arukompas** [comment, marked as answer]
Hey @know-that 

Just add additional include patterns in the `config/log-viewer.php`:

```php
    /**
     * Include file patterns
     */
    'include_files' => [
        '*.log',
        'laravel/**/**/*.log',
    ],
```

**Reply by know-that** [reply]
Referring to your reply, I got the following structure.
![image](https://user-images.githubusercontent.com/31246220/195756558-97eebfa4-e786-4d40-b242-2226eee6cef7.png)
This structure should be more beautiful, just like the directory structure.

**Reply by know-that** [reply]
![image](https://user-images.githubusercontent.com/31246220/195757060-d45b6fa9-abd4-4208-91f4-9c12059536b0.png)
I hope this configuration can read the directory structure
